### PR TITLE
Add Page Preview fields to be shown on the Post Preview Organism

### PR DESCRIPTION
**Pull request by kurtw**
- Changed expandable.html to be a macro and updated affected files
 - The change to the macro was necessary for a future incarnation of the Filterable List Controls

## THIS RELIES ON #1322 
@kave @rosskarchner 

### To Test
- Make sure Page Preview fields save properly. Look for them in the page configuration/settings tab.
- Go to the prototype pages and make sure they're still rendering
- Add an expandable and make sure that renders properly

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/organisms/expandable-group.html [hunk 1]
```diff
@@ -67,10 +67,10 @@
...
     {% if value.expandables %}
-        {% for expandable in value.expandables %}
+        {% for expand in value.expandables %}
             {% if prototype %}
-                {% set value = expandable %}
-                {% include 'molecules/expandable.html' with context %}
+                {% from 'molecules/expandable.html' import expandable with context %}
+                {{ expandable(expand) }}
             {% else %}
                 {{ expandable }}
```
Comment (kave) [inline cfgov/jinja2/v1/_includes/organisms/expandable-group.html]: due to the for loop name change this should be edited it as well

File: cfgov/jinja2/v1/_includes/molecules/expandable.html [hunk 2]
```diff
@@ -85,3 +71,50 @@
...
     </div>
 </div>
+{% endmacro %}
+
+{% if not value %}
+    {% set value = [
+            {
+                'label': 'Default Expandable',
+                'paragraph': lipsum(1, false, 10, 225),
+                'is_bordered': true,
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: Should be `'is_bordered': false,`

File: cfgov/jinja2/v1/_includes/molecules/expandable.html [hunk 3]
```diff
@@ -85,3 +71,50 @@
...
+                'is_midtone': false,
+                'is_expanded': false
+            },
+            {
+                'label': 'Bordered Expandable',
+                'paragraph': lipsum(1, false, 10, 225),
+                'is_bordered': true,
+                'is_midtone': true,
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: Should be `'is_midtone': false,`

File: cfgov/jinja2/v1/_includes/molecules/expandable.html [hunk 4]
```diff
@@ -85,3 +71,50 @@
...
+            },
+            {
+                'label': 'Midtone, Bordered, Expandable - Expanded',
+                'paragraph': lipsum(1, false, 10, 225),
+                'is_bordered': true,
+                'is_midtone': true,
+                'is_expanded': true
+            },
+            {
+                'label': 'Midtone, Bordered, Expandable - Expanded',
```
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/expandable.html]: Should `Midtone, Bordered, Expandable - Expanded` mocked data be listed twice?

**Comment by kave** [comment]
can you add a screenshot of the Preview fields in the admin page.

**Comment by kurtw** [comment]
Closing to be included in #1338

**Comment by kurtw** [comment]
Closing to be included in #1338

**Pull request closed without merging**